config/settings/base.py

```diff
@@ -52,6 +54,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "django_prometheus.middleware.PrometheusAfterMiddleware",
```

Comment:
There's also the database engine, which will monitor database connections/queries, https://github.com/praekeltfoundation/ndoh-hub/blob/develop/ndoh_hub/settings.py#L115-L122 . Not sure if you want it, just making sure that you know about it.

Reply:
Ah right, thanks! Yeah, we did look at this, but TBH Sidekick is mostly acting as a middleman here, so the db requests aren't complicated or heavy.